packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -5039,6 +5084,50 @@ describe("database test", async () => {
           eServiceRiskAnalysisNotFound(eservice.id, riskAnalysisId)
         );
       });
+      it("should throw riskAnalysisDuplicate if risk analysis name is duplicated", async () => {
+        const producerTenantKind: TenantKind = randomArrayItem(
+          Object.values(tenantKind)
+        );
+        const producer: Tenant = {
+          ...getMockTenant(),
+          kind: producerTenantKind,
+        };
+
+        const riskAnalysis_1 = getMockValidRiskAnalysis(producerTenantKind);
+        const riskAnalysis_2 = getMockValidRiskAnalysis(producerTenantKind);
+
+        const eservice: EService = {
+          ...mockEService,
+          producerId: producer.id,
+          mode: eserviceMode.receive,
+          descriptors: [
+            {
+              ...mockDescriptor,
+              state: descriptorState.draft,
+            },
+          ],
+          riskAnalysis: [riskAnalysis_1, riskAnalysis_2],
+        };
+
+        await addOneTenant(producer, tenants);
+        await addOneEService(eservice, postgresDB, eservices);
+
```

Comment:
What about doing the following?
```
const riskAnalysisSeed: EServiceRiskAnalysisSeed = {
  ...buildRiskAnalysisSeed(riskAnalysis_1),
  name: riskAnalysis_2.name,
};
```
In this way we highlight a little more that we are using the same name of `riskAnalysis_2`. If you apply this, you can revert the changes [here](https://github.com/pagopa/interop-be-monorepo/pull/474/files#diff-de383f664ed37e37225728fdb917217369f93604878026d73ba9a1ff491f7761R74-R82).